app/static/src/css/browse.css

```diff
@@ -0,0 +1,236 @@
+.govuk-form-group__search-form {
```

Comment:
Is this file meant to be here? It looks like the compiled version of `app/static/src/scss/browse.scss`.

I would have though it'd been ignored by https://github.com/nationalarchives/da-ayr-beta-webapp/blob/main/.gitignore#L9

Reply:
I've removed this now, all css are in the gitignore 👍